libraries/botbuilder/src/eventFactory.ts

```diff
@@ -30,7 +30,7 @@ export class EventFactory {
         const handoffEvent = this.createHandoffEvent(
             HandoffEventNames.InitiateHandoff,
             handoffContext,
-            context.activity.conversation
+            context.activity.conversation as ConversationAccount // TODO(joshgummersall) why is this necessary?
```

Comment:
The compiler, for whatever reason, refuses to compile without this manual casting. I'm a bit stumped because `context.activity.conversation` is already typed as a `ConversationAccount`.